helm/cluster/files/apps/cluster-autoscaler.yaml

```diff
@@ -9,4 +9,4 @@ dependsOn: kyverno
 namespace: kube-system
 # used by renovate
 # repo: giantswarm/cluster-autoscaler-app
-version: 1.27.3-gs8
+version: 1.27.3-gs9
```

Comment:
Our renovate setup doesn't (currently) update tags w/ prereleases, hence this PR